src/components/v5/common/CompletedAction/partials/CompletedExpenditureContent/CompletedExpenditureContent.tsx

```diff
@@ -64,7 +70,42 @@ const CompletedExpenditureContent: FC<CompletedExpenditureContentProps> = ({
     ],
   } = useActionSidebarContext();
 
-  const { slots = [] } = expenditure;
+  const { slots = [], metadata } = expenditure;
+
+  const stages = (metadata?.stages || []).map((stage) => {
+    const currentSlot = slots.find((slot) => slot.id === stage.slotId);
+    const token = allTokens.find(
+      ({ token: currentToken }) =>
+        currentToken.tokenAddress === currentSlot?.payouts?.[0].tokenAddress,
+    );
+
+    return {
+      milestone: stage.name,
+      amount: moveDecimal(
+        currentSlot?.payouts?.[0].amount,
+        -getTokenDecimalsWithFallback(token?.token.decimals),
+      ),
+      tokenAddress: currentSlot?.payouts?.[0].tokenAddress,
+    };
+  });
+
+  const summedAmount = stages.reduce((acc, { amount, tokenAddress }) => {
+    const token = allTokens.find(
+      ({ token: currentToken }) => currentToken.tokenAddress === tokenAddress,
+    );
+    const formattedAmount = moveDecimal(
+      amount || '0',
+      getTokenDecimalsWithFallback(token?.token.decimals),
```

Comment:
It seems there is no need for this conversion from and then back to wei.

Reply:
Conversions removed